witness/src/Dscp/Witness/Web/Types.hs

```diff
@@ -115,6 +115,7 @@ instance ToJSON TxInfo where
                 [ "txId" .= toTxId tx
                 , "txType" .= ("money" :: Text)
                 , "money" .= tx
+                , "outValue" .= (Coin . sum . map (unCoin . txOutValue) . txOuts $ tx)
```

Comment:
There is `sumCoins`, but ok :man_shrugging:

Reply:
:man_facepalming: